tools/testbench/include/testbench/file.h

```diff
@@ -29,14 +31,15 @@ enum file_format {
 
 /* file component state */
 struct file_state {
-	char *fn;
+	uint64_t cycles_count;
```

Comment:
int32 should be ok, since it is delta between one module. no way to exceed int32

Reply:
As far as I understand this code will only ever run on a GHz 64bits CPU so I don't see what difference  32 bits would make.

Mixing signed and unsigned is fraught with peril and signed overflow is undefined so unsigned is clearly better here.